### ISSUE TITLE
slog: Disabled LogLevel is still being printed

### ISSUE THREAD

**Issue by ccremer** [pterm/pterm#600]
```go
	backend := pterm.DefaultLogger
	logHandler := pterm.NewSlogHandler(backend.WithLevel(pterm.LogLevelDisabled))
	logger.Info("Test")
```

When setting the slog handler level to `LogLevelDisabled`, the message still gets printed.
The bug likely is caused in this if-condition:
https://github.com/pterm/pterm/blob/4048ff9128826714c1c4b39d40c74947355f8dfb/logger.go#L263-L266

In this condition, `l.Level` equals `0`, which is smaller than `3` (3 = Info), so the `return`-clause is skipped and the message gets printed regardless.

**Comment by MarvinJWendt** [comment]
Thanks for reporting the issue! I'll take a look :)

**Comment by MarvinJWendt** [comment]
Released in `v0.12.73` 🚀

**Comment by ccremer** [comment]
That was fast! Thanks 🎉
